style-dictionary/classic/metadata.ts

```diff
@@ -13,6 +13,7 @@ const updatedDescriptions: StyleDictionary.TokenIndex<string> = {
     'The color of clickable elements in their default state. *For example: expandable sections, tabs, and icons.*',
   colorTextInteractiveHover:
     'The color of clickable elements on hover. *For example: expandable sections, and icons on hover.*',
+  colorTextToggleButtonIconPressed: 'The pressed text color of icon toggle buttons.',
```

Comment:
Why does this token have a separate entry in the `classic` style-dictionary?

Reply:
Good catch, we don't need it. Removed.